### ISSUE TITLE
Support for MultiDigraph in Model

### ISSUE THREAD

**Issue by cpmarvin** [tim-fiola/network_traffic_modeler_py3#23]
Hello,

Great work on!!!

Would be great to add support for multiDigraph in the model. This is required to simulate parallel links between nodes.

Regards

**Comment by tim-fiola** [comment]
Acknowledged.  Are parallel links between nodes something you see often?  

I will mock a few things up to determine the scope for this.

Thanks!
Tim

**Comment by cpmarvin** [comment]
hi Tim,

I guess it depends from network to network . Some use LAGs some ECMP. Would be good to support both. 

Regards,

**Comment by tim-fiola** [comment]
Generally, PyNTM is designed to model aggregate capacity between nodes, that is to say, aggregate all capacity of all parallel links between two nodes.  _Note: parallel links means each link connects the same two nodes and each link has the same metric._  I have designed it this way because it improves code performance and because while sometimes I have seen 2 distinct parallel links between two nodes, I have not seen any value in **modeling** them separately.  

If there is a valid use case and/or value to model multiple distinct links with the same metric between two nodes, I definitely want to understand that use case so I can make pyNTM better. 

Or, as another example, if there are 2 distinct links between 2 nodes but they have different metrics, I also want to understand that use case as well.  I'm not sure that'd ever realistically appear in a network, however, so I did not consider that in my initial design.  

Thanks for your help/input here!
